tests/counterparties.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { CreateCounterpartyRequest, Unit } from "../unit"
+
+require("dotenv").config()
+const unit = new Unit(process.env.UNIT_TOKEN || "test", process.env.UNIT_API_URL || "test")
+let counterpartiesId: string[] = []
+
+export function createCounterpartyForTest(id: string) {
+    const req: CreateCounterpartyRequest = {
+        "type": "achCounterparty",
+        "attributes": {
+            "name": "Joe Doe",
+            "routingNumber": "123456789",
+            "accountNumber": "123",
+            "accountType": "Checking",
+            "type": "Person"
+        },
+        "relationships": {
+            "customer": {
+                "data": {
+                    "type": "customer",
+                    "id": id
+                }
+            }
+        }
+    }
+
+    return unit.counterparties.create(req)
+}
+
+describe('Counterparties List', () => {
+    test('Get Counterparties List', async () => {
```

Comment:
don't save counterpartiesId as a state between calls (although it might be faster)
instead in the next tests just call a new function called: GetCounterparties() instead and iterate on it.